internal/pkg/deploy/cloudformation/stack/validate.go

```diff
@@ -463,7 +471,7 @@ func isCorrectSvcNameFormat(s string) bool {
 	return len(trailingMatch) == 0
 }
 
-func validateTopicSubscription(ts manifest.TopicSubscription, validTopicARNs []string) error {
+func validateTopicSubscription(ts manifest.TopicSubscription, validTopicARNs []string, app, env string) error {
```

Comment:
A question here 🙋‍♀️:

When would the `app` and `env` parameters passed into this function be different from the "<app>" and "<env>" component of the valid topic names?

Reply:
If someone tries to add something weird in the manifest. For example, the wanted topic name may have "svc: svc-topicname" and name "bad topic", while the valid topic would have "svc: svc" "name: topic name". This would be a security concern, as the wanted topic needs to match a current valid topic exactly.